src/web/extractors.rs

```diff
@@ -224,6 +224,28 @@ impl FromRequest for BsoBodies {
                 ));
             }
         };
+
+        // ### debug_client
+        if let Some(uids) = &state.limits.debug_clients {
+            for uid in uids {
+                debug!("### checking uaid: {:?}", &uid);
+                let tested = HawkIdentifier::uid_from_path(req.uri(), None).unwrap_or(0);
+                if uid == &tested {
+                    debug!("### returning quota exceeded.");
```

Comment:
This doesn't use any of the benefits of a `HashSet`. This should be changed to call `uids.contains(tested)` instead of looping through all of the IDs in the set.

Reply:
yep. crap.